Makefile.am

```diff
@@ -3879,7 +3879,8 @@ gperf_gperf_sources += \
 libudev_core_la_CFLAGS = \
 	$(AM_CFLAGS) \
 	$(BLKID_CFLAGS) \
-	$(KMOD_CFLAGS)
+	$(KMOD_CFLAGS) \
+        -D UDEV_PROPAGATE_LOG
```

Comment:
i figure all udev related tools should have this set, i.e. ata_id and suchlike, too.